src/main/java/rizzler/AddCommand.java

```diff
@@ -0,0 +1,149 @@
+package rizzler;
+
+import java.time.DateTimeException;
+import java.time.LocalDate;
+
+/**
+ * Represents a command to the Rizzler chatbot to add
+ * a task to the tasklist.
+ *
+ * This class stores the addition of 3 types of tasks:
+ * <code>Todo</code>, <code>Deadline</code>, and <code>Event</code>.
+ */
+public class AddCommand implements Command {
+    private final String taskType;
+    private final String[] fullCommand;
+
+    /**
+     * Constructs the object <code>AddCommand</code>
+     * with the string array parsed by the Parser.
+     *
+     * @param fullCommand Parsed command from the Parser.
+     */
+    AddCommand(String[] fullCommand) {
+        this.taskType = fullCommand[0];
+        this.fullCommand = fullCommand;
+    }
+
+    /**
+     * Executes the command to add the task to the list by
+     * processing the command details into a <code>Task</code>,
+     * then adding it to the TaskList.
+     * @param tasks TaskList supplied by Rizzler.
+     * @param ui Ui supplied by Rizzler.
+     * @param fileStorage FileStorage supplied by Rizzler.
+     * @return String detailing task addition.
+     * @throws RizzlerException If the command was entered incorrectly.
+     */
+    public String execute(TaskList tasks,
+                          Ui ui,
+                          FileStorage fileStorage) throws RizzlerException {
+        switch (this.taskType) {
+        case "todo":
+            if (this.fullCommand.length == 1) {
+                throw new RizzlerException(
+                        "Please add in the task description\n"
+                        + "Format:\n"
+                        + "todo [task name]");
+            } else {
+                String taskName = this.fullCommand[1];
+                for (int i = 2; i < this.fullCommand.length; i++) {
+                    taskName += " " + fullCommand[i];
+                }
+                String output = tasks.add(new Todo(taskName));
+                fileStorage.save(tasks.getListToSave());
+                return output;
+            }
```

Comment:
For switch statement, if a case intended to fall through the next case(no break statement at the end), then you should include //Fallthrough comment at the end for clarity, else you should indicate clearly there's no fall through using break statement. Refer to: [switch statement coding standard](https://se-education.org/guides/conventions/java/intermediate.html)